DPG/Tasks/AOTTrack/MonitorFilterBit.cxx

```diff
@@ -114,45 +114,44 @@ struct CheckFilterBit {
   }
   PROCESS_SWITCH(CheckFilterBit, processData, "process data", true);
 
-    template <typename T> int isFromStrangeDecay(const T& particlesMC,const typename T::iterator& particle){
+  template <typename T>
+  int isFromStrangeDecay(const T& particlesMC, const typename T::iterator& particle)
```

Comment:
minor point: can it be a `bool`? Or are you foreseeing new additional develoments for you you might need more values than `0` or `1`?